docs/intro/quickstart/index.rst

```diff
@@ -0,0 +1,59 @@
+.. _ref_quickstart:
+
+==========
+Quickstart
+==========
+
+.. toctree::
+  :maxdepth: 1
+  :hidden:
+
+  setup
+  modeling
+  working
+  workflow
+  access
+  inheritance
+  dynamic
```

Comment:
We could also have "Summary" section with stuff like: 
- summary of what we build
- "what's next" with links to deployment, auth, ai, etc.
- if we're deploying the quickstart app, we can paste a demo link there
- links to gel examples and templates